bin/cgi_this

```diff
@@ -43,11 +46,15 @@ and runs every script in it as a CGI script.
 
 The following options are available:
 
-=over 4
+=over 5
 
 =item --port PORT
 
-Start the HTTP server ona specific C<PORT>. Default is 3000.
+Start the HTTP server on a specific C<PORT>. Default is 3000.
+
+=item --host IP_ADR
+
+Set HTTP server to listen for incoming connections from C<IP_ADR>. Default is 127.0.0.1.
```

Comment:
You're listening _on_ an address, not from. If you were listening from that address, only incoming connections made by a host with this address would count. That defeats the purpose, doesn't it? :)

You don't need to abbreviate the place holder, it's OK to say `C<IP_ADDRESS>`.